src/guide/typescript/composition-api.md

```diff
@@ -144,10 +144,21 @@ const emit = defineEmits<{
   (e: 'change', id: number): void
   (e: 'update', value: string): void
 }>()
+
+// 3.3+: sintaxis alternativa más concisa
+const emit = defineEmits<{
+  change: [id: number]
+  update: [value: string]
+}>()
 </script>
 ```
 
-El argumento de tipo debe ser un tipo literal con [Call Signatures](https://www.typescriptlang.org/docs/handbook/2/functions.html#call-signatures). El tipo literal se utilizará como el tipo de la función `emit` devuelta. Como podemos ver, la declaración de tipo nos da un control mucho más fino sobre las restricciones de tipo de los eventos emitidos.
+El argumento de tipo puede ser uno de los siguientes:
+
+1. Un tipo de función invocable, pero escrito como un tipo literal con [Call Signatures](https://www.typescriptlang.org/docs/handbook/2/functions.html#call-signatures). Se utilizará como tipo de la función `emit` devuelta.
```

Comment:
Call Signatures -> ¿Firmas de invocación?

Reply:
Si.. sería la traducción apropiada, but should we? Pienso que Call Signatures es como un nombre propio en este caso al tratarse de un concepto core de TypeScript. Que opinan ustedes @glopzel @mrcego?

Reply:
Yo considero traducirlo, aquí hay otro ejemplo: https://apuntes.de/typescript/call-signature/#gsc.tab=0

Reply:
vi unas traducciones y no lo dejan como call signature, la dejan como "firma de función". Cuando busqué fuera de los docs de Vue igual me encontré "firma de llamada", no se cual queda mejor, o la opción "firma de invocación"

Reply:
for the sake of consistency opino que lo dejemos como "Firma de función". Qué opinan?
@mrcego @glopzel

Reply:
Solo tener en cuenta el plural.

Reply:
apoyo "firma de funcion" igual